code/game/mecha/mecha.dm

```diff
@@ -25,7 +25,7 @@
 	force = 5
 	flags = HEAR
 	var/can_move = 1
-	var/mob/living/carbon/occupant = null
+	var/mob/living/carbon/human/occupant = null

```

Comment:
This was accidental(althought the type here really doesn't matter except when trying to grab the DNA).